src/main/java/mate/academy/spring/mapper/impl/response/OrderResponseMapper.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.spring.mapper.impl.response;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderResponseMapper implements DtoResponseMapper<OrderResponseDto, Order> {
+    @Override
+    public OrderResponseDto toDto(Order object) {
+        OrderResponseDto orderResponseDto = new OrderResponseDto();
+        orderResponseDto.setOrderDate(object.getOrderDate());
+        orderResponseDto.setTickets(object.getTickets());
```

Comment:
you should return list of tickets ids, not ticket objects
![image](https://user-images.githubusercontent.com/117590814/219853787-4e904dff-7a07-4dbd-b485-cc1e5a4bd1ae.png)